.github/workflows/blade-chromatic.yml

```diff
@@ -0,0 +1,35 @@
+# Workflow name
+name: Chromatic
+
+# Event for the workflow
+on: push
+
+env:
+  GITHUB_ACCESS_TOKEN: ${{ secrets.CI_BOT_TOKEN }}
+
+# List of jobs
+jobs:
+  chromatic-deployment:
+    name: Chromatic Deployment
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Codebase
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Use Node v14
+        uses: actions/setup-node@v2
+        with:
+          node-version: 14
+      - name: Setup Cache & Install Dependencies
+        uses: bahmutov/npm-install@v1
+        with:
+          install-command: yarn --frozen-lockfile
+      - name: Publish to Chromatic
+        uses: chromaui/action@v1
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Why not use the `CI_BOT_TOKEN` here as well?

Reply:
@kamleshchandnani bumping this. Do we need to use separate tokens in different steps?

Reply:
I'm not using separate token now. I've re-assigned `CI_BOT_TOKEN` to `GITHUB_ACCESS_TOKEN` [here](https://github.com/razorpay/blade/pull/394/files#diff-68c76d748a76d111004464c9f326c495d0be07056640f850ae3a37b3d7661344R5)